### PR TITLE
Add unit test for Keyboard Api's

### PR DESCRIPTION
**Pull request by manoj9788**

### CODE REVIEW

File: test/unit/webdriver/device/keyboard_test.py [hunk 1]
```diff
@@ -32,3 +32,27 @@ def test_hide_keyboard(self):
...
         )
         assert isinstance(driver.hide_keyboard(), WebDriver)
+
+    @httpretty.activate
+    def test_press_keycode(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/press_keycode'),
+            body='{"value": "86"}'
```
Comment (ki4070ma) [inline test/unit/webdriver/device/keyboard_test.py:42]: Here is unnecessary.  Can be deleted.
The return value is `webdriver` instance.
https://github.com/appium/python-client/blob/9e18dfce1ccbe34a5bba3684b054e7f5e22bd229/appium/webdriver/extensions/keyboard.py#L73

File: test/unit/webdriver/device/keyboard_test.py [hunk 2]
```diff
@@ -32,3 +32,27 @@ def test_hide_keyboard(self):
...
+        )
+        driver.press_keycode('86')
+        d = get_httpretty_request_body((httpretty.last_request()))
+        assert d['keycode'] == '86'
```
Comment (ki4070ma) [inline test/unit/webdriver/device/keyboard_test.py:46]: Please add checks for metastate and flags as arguments as well.

https://github.com/appium/python-client/blob/master/appium/webdriver/extensions/keyboard.py#L73

And add another test cases for metastate and flags with values.

File: test/unit/webdriver/device/keyboard_test.py [hunk 3]
```diff
@@ -32,3 +32,27 @@ def test_hide_keyboard(self):
...
+
+    @httpretty.activate
+    def test_press_keycode(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/press_keycode'),
+            body='{"value": "86"}'
+        )
+        driver.press_keycode('86')
```
Comment (ki4070ma) [inline test/unit/webdriver/device/keyboard_test.py:44]: Check the return value.
And arg is `int`.

https://github.com/appium/python-client/blob/9e18dfce1ccbe34a5bba3684b054e7f5e22bd229/appium/webdriver/extensions/keyboard.py#L79

File: test/unit/webdriver/device/keyboard_test.py [hunk 4]
```diff
@@ -32,3 +32,27 @@ def test_hide_keyboard(self):
             appium_command('/session/1234567890/appium/device/hide_keyboard')
         )
         assert isinstance(driver.hide_keyboard(), WebDriver)
+
```
Comment (ki4070ma) [inline test/unit/webdriver/device/keyboard_test.py:35]: Please add test cases for other Keyboard Apis, not only below 2 apis.

File: test/unit/webdriver/device/keyboard_test.py [hunk 5]
```diff
@@ -32,3 +32,27 @@ def test_hide_keyboard(self):
...
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/press_keycode'),
+            body='{"value": "86"}'
+        )
+        driver.press_keycode('86')
+        d = get_httpretty_request_body((httpretty.last_request()))
+        assert d['keycode'] == '86'
+
+    @httpretty.activate
+    def test_long_press_keycode(self):
```
Comment (ki4070ma) [inline test/unit/webdriver/device/keyboard_test.py:49]: Please follow above comments as well.

File: test/unit/webdriver/device/keyboard_test.py (hunk 4 above)
Comment (manoj9788) [reply test/unit/webdriver/device/keyboard_test.py:35]: I saw somewhere KeyEvent is Selendroid - So didn't add it!
What else?

Comment (ki4070ma) [reply test/unit/webdriver/device/keyboard_test.py:35]: You can see other 3 apis in below file.
https://github.com/appium/python-client/blob/9e18dfce1ccbe34a5bba3684b054e7f5e22bd229/appium/webdriver/extensions/keyboard.py

```
is_keyboard_shown
hide_keyboard
keyevent
```

Comment (manoj9788) [reply test/unit/webdriver/device/keyboard_test.py:35]: will add tests for metastate and flag args

**Pull request closed without merging**